codegen/smithy-ruby-codegen-test/integration-specs/errors_spec.rb

```diff
@@ -64,6 +64,16 @@ module Errors
         expect(error).to be_a(ApiClientError)
         expect(error.message).to eq('error message')
       end
+
+      it 'is retryable without throttling' do
+        http_resp = Hearth::HTTP::Response.new
+        data = Types::ClientError.new
+        # don't test fakeProtocol parsers
+        expect(Parsers::ClientError).to receive(:parse).with(http_resp).and_return(data)
```

Comment:
This is outside the scope of this PR - but I'm a bit concerned we can't initialize our errors without them parsing.